test/TestProjects/NameConflicts/Generated/Operations/ServiceClient.cs

```diff
@@ -55,5 +58,85 @@ public virtual Response<Struct> Operation(string request, string message, string
         }
+
+        /// <summary> Analyze body, that could be different media types. </summary>
+        /// <param name="originalResponse"> The original response from starting the operation. </param>
+        /// <param name="createOriginalHttpMessage"> Creates the HTTP message used for the original request. </param>
+        internal Operation<Response> CreateAnalyzeBody(Response originalResponse, Func<HttpMessage> createOriginalHttpMessage)
+        {
+            if (originalResponse == null)
+            {
+                throw new ArgumentNullException(nameof(originalResponse));
+            }
+            if (createOriginalHttpMessage == null)
+            {
+                throw new ArgumentNullException(nameof(createOriginalHttpMessage));
+            }
+
+            return ArmOperationHelpers.Create(_pipeline, _clientDiagnostics, originalResponse, RequestMethod.Post, "ServiceClient.AnalyzeBody", OperationFinalStateVia.Location, createOriginalHttpMessage);
+        }
+
+        /// <summary> Analyze body, that could be different media types. </summary>
+        /// <param name="stringBody"> The binary to use. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        public virtual async ValueTask<Operation<Response>> StartAnalyzeBodyAsync(Stream stringBody, CancellationToken cancellationToken = default)
+        {
+            if (stringBody == null)
+            {
+                throw new ArgumentNullException(nameof(stringBody));
+            }
+
+            var originalResponse = await RestClient.AnalyzeBodyAsync(stringBody, cancellationToken).ConfigureAwait(false);
+            return CreateAnalyzeBody(originalResponse, () => RestClient.CreateAnalyzeBodyRequest(stringBody));
+        }
+
+        /// <summary> Analyze body, that could be different media types. </summary>
+        /// <param name="stringBody"> The binary to use. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        public virtual Operation<Response> StartAnalyzeBody(Stream stringBody, CancellationToken cancellationToken = default)
+        {
+            if (stringBody == null)
+            {
+                throw new ArgumentNullException(nameof(stringBody));
+            }
+
+            var originalResponse = RestClient.AnalyzeBody(stringBody, cancellationToken);
+            return CreateAnalyzeBody(originalResponse, () => RestClient.CreateAnalyzeBodyRequest(stringBody));
+        }
+
+        /// <summary> Analyze body, that could be different media types. </summary>
+        /// <param name="originalResponse"> The original response from starting the operation. </param>
+        /// <param name="createOriginalHttpMessage"> Creates the HTTP message used for the original request. </param>
+        internal Operation<Response> CreateAnalyzeBody0(Response originalResponse, Func<HttpMessage> createOriginalHttpMessage)
```

Comment:
The result.

Reply:
I'm a bit confused. The code and parameters for this are identical to `CreateAnalyzeBody`. These Create methods (for the same operation) would always be the same?? The difference in `StartAnalyzeBody` is based on Content-Type, which only applies to body parameters. Since body parameters don't apply here, when would the `Create` methods differ?

It sounds like, based on the issue, that we'd have the right `StartAnalyzeBody` overloads, but not duplicate `CreateAnalyzeBody` methods.

Reply:
Good point, deduplication might be better.